### ISSUE TITLE
Define the python version

### ISSUE THREAD

**Issue by eulovi** [imcs-compsim/meshpy#76]
While trying to run Meshpy, I noticed that Meshpy is now using Python 3.12. In the Installation instructions, it is unclear if this Python version is used, since with `python3 -m venv meshpy-env` a python environment is created with the local Python3 version, which can be different from Python 3.12.   
Should the installation instructions be updated?

**Comment by davidrudlstorfer** [comment]
It would be nice to just switch to a conda environment and maybe update the install instructions accordingly?

Something like

`conda create -n meshpy python=3.12`

`conda activate meshpy`

`pip install -e .`

**Comment by isteinbrecher** [comment]
Thanks for the suggestion. We should definitely adapt the script so it works with the `main` version of `meshpy`.
@davidrudlstorfer I am against an installation instruction completely relying on `conda`. But maybe something like:


## Installation

MeshPy is developed with `python3.12`. Other versions of Python might lead to issues. It is recommended to use a python environment container such as `venv` or `conda`.
- `venv`: Chose an appropriate directory for this, e.g., `/home/user/opt`. A virtual environment can be created with (make sure that python3 points to a python3.12
  ```bash
  python3 -m venv meshpy-env
  ```
  The created virtual environment can be loaded with
  ```bash
  source meshpy-env/bin/activate
  ```
- `conda`: A condo environment can be created with
  ```
  conda create -n meshpy python=3.12
  conda activate meshpy
  ```

From now on we assume that the previously created enviroment is loaded. To install `meshpy` go to the repository root directory
```bash
cd <path_to_meshpy>
```
...

**Comment by isteinbrecher** [comment]
Closed with #109
